htlcswitch/link_test.go

```diff
@@ -1487,9 +1487,10 @@ func newSingleLinkTestHarness(chanAmt, chanReserve btcutil.Amount) (
 		BlockEpochs:    globalEpoch,
 		BatchTicker:    ticker,
 		FwdPkgGCTicker: NewBatchTicker(time.NewTicker(5 * time.Second)),
-		// Make the BatchSize large enough to not
-		// trigger commit update automatically during tests.
-		BatchSize: 10000,
+		// Make the BatchSize and UpdateFeeInterval large enough to not
+		// trigger commit updates automatically during tests.
+		BatchSize:         10000,
+		UpdateFeeInterval: func() time.Duration { return time.Hour },
```

Comment:
Can you use the `BatchTicker` pattern to control the ticks instead?

Reply:
The `BatchTicker` seems to be of better use when wanting continuous ticks within the same interval, rather than a different interval after every tick.